From 0dfeb49aa061641a87964484318fdc6c661f4429 Mon Sep 17 00:00:00 2001
From: Lily Zhang <r58066@freescale.com>
Date: Fri, 9 Mar 2012 21:48:57 +0800
Subject: ENGR00176812-7 mx6 sabreauto: remove MX6Q_PAD_EIM_EB2__GPIO_2_30

- EIM_EB2 is used as I2C2_SCL instead of GPIO for SPI
NOR for RevA board
- Use CONFIG_MTD_M25P80 to control SPI NOR code only

Signed-off-by: Lily Zhang <r58066@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabreauto.h    |    3 +--
 arch/arm/mach-mx6/board-mx6solo_sabreauto.h |    6 +++---
 2 files changed, 4 insertions(+), 5 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabreauto.h b/arch/arm/mach-mx6/board-mx6q_sabreauto.h
index 74b1c57..9ff0a36 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabreauto.h
+++ b/arch/arm/mach-mx6/board-mx6q_sabreauto.h
@@ -188,7 +188,6 @@ static iomux_v3_cfg_t mx6q_sabreauto_pads[] = {
 	MX6Q_PAD_GPIO_8__UART2_RXD,
 	MX6Q_PAD_SD4_DAT6__UART2_CTS,
 	MX6Q_PAD_SD4_DAT5__UART2_RTS,
-#if defined(CONFIG_MTD_M25P80) || defined(CONFIG_MTD_M25P80_MODULE)
 	/*USBs OC pin */
 	MX6Q_PAD_EIM_WAIT__GPIO_5_0,  /*HOST1_OC*/
 	MX6Q_PAD_SD4_DAT0__GPIO_2_8,  /*OTG_OC*/
@@ -209,12 +208,12 @@ static iomux_v3_cfg_t mx6q_sabreauto_pads[] = {
 	MX6Q_PAD_GPIO_19__GPIO_4_5,
 	MX6Q_PAD_EIM_D24__GPIO_3_24,
 
+#if defined(CONFIG_MTD_M25P80) || defined(CONFIG_MTD_M25P80_MODULE)
 	/* eCSPI1 */
 	MX6Q_PAD_EIM_D16__ECSPI1_SCLK,
 	MX6Q_PAD_EIM_D17__ECSPI1_MISO,
 	MX6Q_PAD_EIM_D18__ECSPI1_MOSI,
 	MX6Q_PAD_EIM_D19__ECSPI1_SS1,
-	MX6Q_PAD_EIM_EB2__GPIO_2_30,	/*SS0*/
 	MX6Q_PAD_EIM_D19__GPIO_3_19,	/*SS1*/
 #else
 	/* Parallel NOR */
diff --git a/arch/arm/mach-mx6/board-mx6solo_sabreauto.h b/arch/arm/mach-mx6/board-mx6solo_sabreauto.h
index b507766e..360ec9e 100644
--- a/arch/arm/mach-mx6/board-mx6solo_sabreauto.h
+++ b/arch/arm/mach-mx6/board-mx6solo_sabreauto.h
@@ -189,7 +189,6 @@ static iomux_v3_cfg_t mx6dl_sabreauto_pads[] = {
 	MX6DL_PAD_GPIO_8__UART2_RXD,
 	MX6DL_PAD_SD4_DAT6__UART2_CTS,
 	MX6DL_PAD_SD4_DAT5__UART2_RTS,
-#if defined(CONFIG_MTD_M25P80) || defined(CONFIG_MTD_M25P80_MODULE)
 	/*USBs OC pin */
 	MX6DL_PAD_EIM_WAIT__GPIO_5_0,  /*HOST1_OC*/
 	MX6DL_PAD_SD4_DAT0__GPIO_2_8,  /*OTG_OC*/
@@ -210,13 +209,14 @@ static iomux_v3_cfg_t mx6dl_sabreauto_pads[] = {
 	MX6DL_PAD_GPIO_19__GPIO_4_5,
 	MX6DL_PAD_EIM_D24__GPIO_3_24,
 
+#if defined(CONFIG_MTD_M25P80) || defined(CONFIG_MTD_M25P80_MODULE)
 	/* eCSPI1 */
 	MX6DL_PAD_EIM_D16__ECSPI1_SCLK,
 	MX6DL_PAD_EIM_D17__ECSPI1_MISO,
 	MX6DL_PAD_EIM_D18__ECSPI1_MOSI,
 	MX6DL_PAD_EIM_D19__ECSPI1_SS1,
-	MX6DL_PAD_EIM_EB2__GPIO_2_30,	/*SS0*/
-	MX6DL_PAD_EIM_D19__GPIO_3_19,	/*SS1*/
+
+	MX6DL_PAD_EIM_D19__GPIO_3_19,
 #else
 	/* Parallel NOR */
 	MX6DL_PAD_EIM_OE__WEIM_WEIM_OE,
-- 
1.7.9.5

